index.php

```diff
@@ -902,12 +902,12 @@ function drawmap() {
 
                 // Elevation Profile
                 layer_elevation_profile_contours = new OpenLayers.Layer.TMS(
-                    'ASTER GDEM Contour Lines (zoom 13-17)',
-                    'http://korona.geog.uni-heidelberg.de/tiles/asterc/?',
+                    'MapSurfer ASTER GDEM contour lines (layer 13-17)',
+                    'http://maps.heigit.org/openmapsurfer/tiles/asterc/webmercator/',
```

Comment:
```suggestion
                    'https://maps.heigit.org/openmapsurfer/tiles/asterc/webmercator/',
```

Is there something agains using HTTPS? The map should load as much data as possible via https because it shoudl be served via HTTPS as well.

Reply:
Good catch. I have checked and https works as well on that server. At least not any worse than http -- I do get a lot of error 503 on that server, it seems a bit overloaded.